static/app/utils/replays/getCurrentScreenName.tsx

```diff
@@ -14,6 +16,10 @@ function getCurrentScreenName(
     return '';
   }
 
+  if ('category' in mostRecentFrame && mostRecentFrame.category === 'replay.init') {
+    return replayRecord?.urls[0] ?? '';
+  }
```

Comment:
idk if this is needed too? 

```suggestion
  if (repalyRecord && 'category' in mostRecentFrame && mostRecentFrame.category === 'replay.init') {
    return replayRecord?.urls[0] ?? '';
  }
```

Reply:
this is what's in `getCurrentUrl.tsx`, which doesn't have that check:
https://github.com/getsentry/sentry/blob/15997098796a3d3b1297227e80b6dd06ab5a461b/static/app/utils/replays/getCurrentUrl.tsx#L28-L30

Reply:
but that snippet isn't reading from `replayRecord?.urls`

There's probably never a case where `replayRecord` is undefined, and `mostRecentFrame` has some useful value, I was just thinking through the typesystem. nbd!